### DISCUSSION TITLE
For "Configure Tasks..." what is needed to have detected compiler template options be avaliable?

### DISCUSSION THREAD

**Discussion by mofosyne** [VSCodium/vscodium#1772] [Q&A]
Hey, I was trying to attempt to follow along this tutorial https://www.youtube.com/watch?v=8jLOx1hD3_o on C++ programming with vscodium and it's proving quite difficult to get the build system going for it. So far I'm trying to use `llvm-vs-code-extensions.vscode-clangd` and C++ code runner extention as an alternative to microsoft intellisense.

One thing I noticed however is that I can never get the detected task feature in `Terminal > create tasks.json from template` to show something like

![image](https://github.com/VSCodium/vscodium/assets/827793/e713ee50-50d7-4d20-8222-e5ddf1f2ef2b)

Instead I'm getting 

![image](https://github.com/VSCodium/vscodium/assets/827793/b8d1c2c2-78d9-499d-833f-31f6c15c3eb7)

I'm a bit stumped but would really like to be able to use vscodium when doing C++ specific coding so advice would be appreciated on steps, or if a bug ticket should be issued

**Comment by RavenRandomz** [comment]
Whelp
